common/interpreter/formatSwitch_test.go

```diff
@@ -3,6 +3,8 @@ package interpreter
 import (
 	"errors"
 	"testing"
+
+	"github.com/amp-labs/connectors/test/utils/testutils"
 )
 
 func TestFormatSwitchParseJSON(t *testing.T) { //nolint:funlen
```

Comment:
I think we should keep the json cases as well as the list of json cases in test cases so that we can cover. I think this PR enables the ParseJSON to be able to parse both cases.

Reply:
I am not understanding this part:
> in **test cases** so that we **can cover**.

Do you mean that there should be a separate function to make a distinction? Should I create a new test suite with one test case for array?

ParseJSON indeed can parse either array or json object, thats why I added it under one test suite. Array input is just one of a special cases for ParseJSON.

Reply:
nvm, I misinterpreted your code in test.